### ISSUE TITLE
Airbytes does not cast correctly the types of Airtable columns

### ISSUE THREAD

**Issue by csauge** [airbytehq/airbyte#18063]
## Environment
- **Airbyte version**:  v0.40.14
- **OS Version / Instance**: macOS Monterey v12.6 MacBookAir M1 2020 16 GB
- **Deployment**: Docker
- **Source Connector and version**: Airtable 0.1.2
- **Destination Connector and version**: Postgres 0.3.26
- **Step where error happened**: Deploy

## Current Behavior
When selecting a Airtable stream as a source and Postgres as a destination, any column with a Number type in Airtable is mapped as a string in _airbyte_data JSON blob and text in the postgres table after normalization

## Expected Behavior
Any Number type Airtable column type whould be mapped as an integer (or similar type) in postgres. This seems applicable also to  all other types

## Steps to Reproduce
1. Get data from a Airtable table with at least one Number-type column as a source 
2. Select postgres as destination
3. Postgres table before and after normalization contains only text-type column (except the airbyte added columns)

## Are you willing to submit a PR?
Yes

**Comment by bazarnov** [comment]
This is a known limitation as for now, the connector is still in beta. Now the connector treats all the data as `string`, we will add the support of correct casting in the future.

**Comment by YowanR** [comment]
@bazarnov Can you explain why this is a limitation and we default to all types as string? Do other connectors not support casting? Just trying to gain more context from your comment here.

**Comment by bazarnov** [comment]
@YowanR 

No, it was not a scope for a beta certification, but will be added into GA, or we can prioritize this one for next sprint and add native casting for better schema translation.
